moped-editor/src/views/projects/projectView/ProjectComponents/mapUtils.js

```diff
@@ -171,11 +186,22 @@ export const ProjectComponentsSourcesAndLayers = ({
       >
         <Layer
           beforeId="street-labels"
-          {...mapStyles[
-            clickedComponent || isEditingComponent
-              ? "project-points-muted"
-              : "project-points"
-          ].layerProps}
```

Comment:
Another conditional render that is now split into two layers with separated visibility logic